src/test/Makefile-client.am

```diff
@@ -200,7 +200,7 @@ bin_DEBUGPROGRAMS += ceph_test_cls_numops
 ceph_test_cls_journal_SOURCES = test/cls_journal/test_cls_journal.cc
 ceph_test_cls_journal_LDADD = \
         libcls_journal_client.la $(LIBRADOS) \
-        $(LIBCOMMON) $(CRYPTO_LIBS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD) -luuid
+        $(LIBCOMMON) $(CRYPTO_LIBS) $(UNITTEST_LDADD) $(RADOS_TEST_LDADD)
```

Comment:
@ukernel 
Oke, I see, Will check if the FreeBSD Boost package has UUID stuff in it, since not all Boost is packaged for FreeBSD.